heaps/min_heap.py

```diff
@@ -57,18 +75,33 @@ def heap_up(self, index):
             property is reestablished.
             
             This could be **very** helpful for the add method.
-            Time complexity: ?
-            Space complexity: ?
+            Time complexity: o(1)
+            Space complexity: o(1)
```

Comment:
✨ Creative to use this in conjunction with iteration in your `add` function. For consistency style wise, I would recommend sticking to either an iterative or recursive solution between `heap_up` and `heap_down`. _Technically_ the specification does say `heap_up` should perform its operation "until the Heap property is reestablished" which indicates recursion, but I'm not too concerned with that.